src/Components/Login.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { ReactDOM } from "react";
+const App = (props) => <LoginForm />;
+
+class LoginForm extends React.Component {
```

Comment:
In my opinion, It is better to use functional components. Read this to know more -> https://stackoverflow.com/a/57579403